tokio-trace-fmt/src/filter.rs

```diff
@@ -96,8 +102,10 @@ where
 
 impl Filter for EnvFilter {
     fn callsite_enabled(&self, metadata: &Metadata, _: &span::Context) -> Interest {
-        if metadata.level() > &self.max_level {
-            return Interest::never();
+        if !self.includes_span_directive {
+            if metadata.level() > &self.max_level {
```

Comment:
Nit: I think this would read better if instead of the nested `if`s we just had a single one, like:
```rust
if !self.includes_span_directive && metadata.level() > &self.max_level {
    return Interest::never();
}
```